bitnami/redis/values.yaml

```diff
@@ -1959,6 +1965,41 @@ volumePermissions:
     seLinuxOptions: null
     runAsUser: 0
 
+## Kubectl InitContainer
+## used by Sentinel to update the isMaster label on the Redis(TM) pods
+##
+kubectl:
+  ## Bitnami Kubectl image version
+  ## ref: https://hub.docker.com/r/bitnami/kubectl/tags/
+  ## @param kubectl.image.registry [default: REGISTRY_NAME] Kubectl image registry
+  ## @param kubectl.image.repository [default: REPOSITORY_NAME/kubectl] Kubectl image repository
+  ## @skip kubectl.image.tag Kubectl image tag (immutable tags are recommended), by default, using the current version
+  ## @param kubectl.image.digest Kubectl image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
+  ## @param kubectl.image.pullPolicy Kubectl image pull policy
+  ## @param kubectl.image.pullSecrets Kubectl pull secrets
+  ##
+  image:
+    registry: docker.io
+    repository: bitnami/kubectl
+    tag: {{ .Capabilities.KubeVersion }}
```

Comment:
This is error-prone, we should stick to a specific version as we do in other charts, see:

- https://github.com/bitnami/charts/blob/main/bitnami/mongodb/values.yaml#L847

It'd be the user responsibility to adapt the kubectl image version to use if required.